python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md

```diff
@@ -37,7 +39,7 @@ pip3 install -r requirements.txt
 
 ## Running the ONNXRuntime OpenVINO Execution Provider sample
 ```bash
-python3 tiny_yolov2_obj_detection_sample.py
+python3 tiny_yolov2_obj_detection_sample.py --video bottle-detection.mp4 --model tinyyolov2.onnx
```

Comment:
Can we also make passing device type options configurable with the sample.
Default, it should pick OpenVINO-EP CPU.
But the user should also have options to run on other device_types in OpenVINO-EP and also MLAS (default CPU-EP)

Reply:
Resolved